CONTRIBUTING.md

```diff
@@ -94,6 +94,16 @@ bazel test -c opt -k \
 --test_output=errors \
 //tensorflow_addons/...
 ```
+This test will test Python2 by default.
```

Comment:
Since Python2 is being sunset in favor of Python3, @seanpmorgan @facaiy @Smokrow What is your opinion on making Python3 testing by default ?

Reply:
Hi, Armando, I think we test both py2 and py3. It seems not necessary to make a decision here, which depends on the user's environment

Reply:
Yeah until 2020 we have to support and test both.